scripts/prove/bin/multi.rs

```diff
@@ -113,37 +100,32 @@ async fn main() -> Result<()> {
             ))
             .expect("saving proof failed");
     } else {
-        let start_time = Instant::now();
-        let (_, report) = prover
-            .execute(MULTI_BLOCK_ELF, sp1_stdin.clone())
-            .run()
-            .unwrap();
-        let execution_duration = start_time.elapsed();
-
-        let l2_chain_id = data_fetcher.get_l2_chain_id().await.unwrap();
-        let report_path = format!(
-            "execution-reports/multi/{}/{}-{}.csv",
-            l2_chain_id, l2_start_block, l2_end_block
-        );
-
-        // Create the report directory if it doesn't exist.
-        let report_dir = format!("execution-reports/multi/{}", l2_chain_id);
-        if !std::path::Path::new(&report_dir).exists() {
-            fs::create_dir_all(&report_dir).unwrap();
-        }
-
-        let block_data = data_fetcher
-            .get_l2_block_data_range(l2_start_block, l2_end_block)
-            .await?;
```

Comment:
Why did you remove this? We need to create the directory unless it already exists here.

Reply:
nvm